app/src/main/java/chat/rocket/android/contacts/ContactsFragment.kt

```diff
@@ -2,31 +2,60 @@ package chat.rocket.android.contacts
 
 import android.Manifest
 import android.content.pm.PackageManager
+import android.graphics.Color
 import android.os.Bundle
-import android.provider.ContactsContract
 import android.view.*
+import android.widget.ImageView
 import androidx.appcompat.app.AppCompatActivity
 import androidx.appcompat.widget.SearchView
 import androidx.core.content.ContextCompat
 import androidx.fragment.app.Fragment
 import chat.rocket.android.R
 import chat.rocket.android.contacts.models.Contact
-import chat.rocket.android.createchannel.ui.CreateChannelFragment
 import chat.rocket.android.main.ui.MainActivity
 import chat.rocket.android.util.extension.onQueryTextListener
 import kotlinx.android.synthetic.main.app_bar.*
 import java.util.ArrayList
-import kotlin.Comparator
 import kotlin.collections.HashMap
 
 // WIDECHAT
 import chat.rocket.android.helper.Constants
 import com.facebook.drawee.view.SimpleDraweeView
+import android.view.LayoutInflater
+import android.widget.TextView
+import androidx.lifecycle.ViewModelProviders
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import chat.rocket.android.chatrooms.adapter.ItemHolder
+import chat.rocket.android.chatrooms.viewmodel.ChatRoomsViewModel
+import chat.rocket.android.chatrooms.viewmodel.Query
+import chat.rocket.android.contacts.adapter.ContactHeaderItemHolder
+import chat.rocket.android.contacts.adapter.ContactItemHolder
+import chat.rocket.android.contacts.adapter.ContactRecyclerViewAdapter
+import chat.rocket.android.contacts.adapter.inviteItemHolder
+import chat.rocket.android.db.DatabaseManagerFactory
+import chat.rocket.android.server.domain.GetCurrentServerInteractor
+import chat.rocket.android.util.extensions.avatarUrl
+import dagger.android.support.AndroidSupportInjection
+import io.reactivex.Single
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.rxkotlin.subscribeBy
+import io.reactivex.schedulers.Schedulers
+import javax.inject.Inject
+
+
 
 /**
  * Load a list of contacts in a recycler view
  */
 class ContactsFragment : Fragment() {
+    @Inject
```

Comment:
@kb0304 I'm putting this comment here because it's not a part of the current PR, but I'm wondering why we need this method....  populateContacts()  https://github.com/WideChat/Rocket.Chat.Android/pull/176/files#diff-1c45a7b01d732d24e11572fc5667deaaR211

Why not just call the getContactList() method directly when you need it?

Reply:
yes in current PR we dont need popular contacts i can remove it it was on added to add public contacts